rubrik_cdm/physical.py

```diff
@@ -37,7 +37,9 @@ def add_physical_host(self, hostname, timeout=60):
             str -- No change requird. The host '`hostname`' is already connected to the Rubrik cluster.
             dict -- The full API response for `POST /v1/host`.
         """
-        if(len(hostname) == 0):
+        count_of_hosts = len(hostname)
+
+        if(count_of_hosts == 0):
```

Comment:
We want to throw an error message if this list is empty so something like

```python
if count_of_hosts == 0:
   raise InvalidParameterException("The provided hostname list is empty")
```
The theory being that the list should always be populated and if its not something is wrong so throw an error. 

Generally you see those checks through the SDK as a standalone if statement as well.

Reply:
Fixed in next commit.